### PR TITLE
Update to user tutorial

### PR DESCRIPTION
**Pull request by KennellyT**
The previous Cyclus user tutorial used cyclist to teach users how to write Cyclus input files. As noted by this [git issue](https://github.com/cyclus/cyclus.github.com/issues/267), a more complete guide for
creating Cyclus input files with an intermediate tutorial on [separations](https://github.com/cyclus/cyclus.github.com/issues/256) was needed. This pull request deletes mention and usage of cyclist in the tutorial and restructures the website's Cyclus tutorial (`section/user/tutorial`) to teach users how to write a XML Cyclus input file, run Cyclus via Cycrun or natively on your computer, and conduct analysis on the output file. I did not know if the solution to  [`Getting Started on Recycle`](http://fuelcycle.org/user/tutorial/add_arche_commod_recipe.html) should be put on the website so I put it in the [arfc/cyclus-tutorial](https://github.com/arfc/cyclus-tutorial/pull/7) repository.

### CODE REVIEW

File: source/user/tutorial/cyclist_tour.rst [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    * Data Exploration: to explore the output of a particular scenario or set of scnearios
-* A set of **tools** that can be imported into the workspace
-* A **message window** in which Cyclist may occasionally report updates.
+* A **Run** button that runs the cell you're in
+* A **Up and Down** buttons that move the cell up or down
+* A vertical blue line that shows what cell you're currently active in
+* A **Stop** button that stops running the cell you're in
 
-.. image:: cyclist_tour_annotated.png
+.. image:: ipython_.png
```
Comment (bam241) [inline source/user/tutorial/cyclist_tour.rst]: this image does not exist. maybe `ipython_tour.png`

File: source/user/tutorial/cyclus_tour.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+=========================================================
+
+An IPython notebook has four main areas:
+
+* A **Run** button that runs the cell you're in
+* A **Up and Down** buttons that move the cell up or down
+* A vertical blue line that shows what cell you're currently active in
+* A **Stop** button that stops running the cell you're in
+
+.. image:: ipython_.png
```
Comment (bam241) [inline source/user/tutorial/cyclus_tour.rst]: image does not exist.

File: source/user/tutorial/ipython_tour.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+=========================================================
+
+An IPython notebook has four main areas:
+
+* A **Run** button that runs the cell you're in
+* A **Up and Down** buttons that move you up or down a cell
+* A vertical blue line that shows what cell you're currently in
+* A **Stop** button that stops running the cell you're in
+
+.. image:: ipython_.png
```
Comment (bam241) [inline source/user/tutorial/ipython_tour.rst]: image does not exist.

File: source/user/tutorial/launch_cyclus.rst [hunk 4]
```diff
@@ -7,22 +7,24 @@ Choosing Where to Run
...
 ++++++++++++++++++++++++++++++++++++++++++++
 
-1. Select the Server: http://cycrun.fuelcycle.org
-2. Click the "Execute" button
-3. This will launch a job and show you the status in the "Jobs" panel.
+1. Find the your input file (.xml) on your computer
+2. Go to the Server: http://cycrun.fuelcycle.org
+3. Copy and paste the file's information into the window
+3. Click the "Submit" button
+4. This will launch a job and show you the "Job Id"
```
Comment (bam241) [inline source/user/tutorial/launch_cyclus.rst]: numbering issue (3 occurs 2 time)

File: source/user/tutorial/run_cyclus_native.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
+Working with Cyclus in on your machine
+======================================
+
+Choosing Where to Run
+-----------------------
```
Comment (bam241) [inline source/user/tutorial/run_cyclus_native.rst]: underline too long

File: source/user/tutorial/run_cyclus_native.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+For today's tutorial, CYCLUS has been install natively in the cloud so it can be
+run from either the IPython notebook or in the command prompt.
+
+.. image:: cyclus_in_IP.png
+    :align: center
+    :alt: Running CYCLUS in an IPython Notebook
+
+Activity: Execute your Scenario in an IPython Notebook
+++++++++++++++++++++++++++++++++++++++++++++
```
Comment (bam241) [inline source/user/tutorial/run_cyclus_native.rst]: underline too small (should match exactly the title)

File: source/user/tutorial/run_cyclus_native.rst [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+2. Remove any old cyclus output files by: ``!rm tutorial.sqlite``
+3. Run CYCLUS by: ``!cyclus input.xml -o tutorial.sqlite``
+
+
+.. image:: cyclus_run.png
+    :align: center
+    :alt: Job status is shown when executing CYCLUS
+
+Activity: Retrieve your Results for Analysis
+++++++++++++++++++++++++++++++++++++++++++++++
```
Comment (bam241) [inline source/user/tutorial/run_cyclus_native.rst]: title underline too long

File: source/user/tutorial/run_cyclus_native.rst [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+For today's tutorial, CYCLUS has been install natively in the cloud so it can be
+run from either the IPython notebook or in the command prompt.
+
+.. image:: cyclus_in_IP.png
+    :align: center
+    :alt: Running CYCLUS in an IPython Notebook
+
+Activity: Execute your Scenario in an IPython Notebook
+++++++++++++++++++++++++++++++++++++++++++++
+1. Go to the IPython notebook
```
Comment (bam241) [inline source/user/tutorial/run_cyclus_native.rst]: should add an empty line between the title and the text
(my advise is to add 2 blank lines before a title and one after, to improve the readability)

**Comment by KennellyT** [comment]
Thank you for the helpful review @bam241 - I have fixed typos, link issues, and title underline issues in the updated website. I am adding clarity to the beginning steps of the tutorial on the website and I will roll this out tomorrow morning!

**Comment by KennellyT** [comment]
The best way to review and finalized the tutorial would be to break it up and make the changes discussed and listed the google doc file. 
The main files of the tutorial are listed below and I have assigned a few files per person to spread out the work load. 
Exercise 1:
    install_launch_cyclus - Tyler
    ipython_tour -Tyler
    sim_parm -Tyler
    add_arche -Tyler
    add_proto- @jbae11 
    add_reg_inst - @jbae11
    launch_cyclus - @gwenchee 
    run_cyclus_native -  @gwenchee 
    data_explorer - @gtw2 
    add_second_reactor- @gtw2 

Exercise 2 is a lot shorter than exercise 1 as it has not been changed from the current fuelcycle.org tutorial and is meant to be a more "hands off" tutorial. 
Exercise 2:
    add_arche_commod_recipe - Tyler
    add_sep - Tyler
    add_fab - @jbae11 
    mod_rxtr- @gwenchee 
    add_deploy - @gtw2

**Comment by jbae11** [comment]
@KennellyT  Thanks for putting this together! Just to be clear, we'd make the edits and make a PR to `KennellyT:kennellyt_simple` not `arfc:source` right? Meaning that this PR will not be merged until our PRs get merged? (@katyhuff)

**Comment by katyhuff** [comment]
@jbae11 quite the opposite. I'd like to merge this now and let @KennellyT mostly off the hook for now. Each of the things identified in the meeting should be made into issues. You can assign yourselves to them and address them in *separate* *small* pull requests.

**Pull request merged**